Point_set_processing_3/examples/Point_set_processing_3/CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ if ( CGAL_FOUND )
     find_package(libpointmatcher QUIET)
     include(CGAL_pointmatcher_support)
     if (TARGET CGAL::pointmatcher_support)
+      find_package(Boost COMPONENTS thread filesystem system program_options date_time chrono REQUIRED)
```

Comment:
What is that `find_package(Boost ...)` for? It does not seem used in the following lines.

Reply:
libpointmatcher forwards all its boost dependencies, and those are the said dependencies. Without this find_package, cmake generation fails with 
```cmake 
CMake Error at CMakeLists.txt:117 (add_executable):
  Target "registration_with_opengr_pointmatcher_pipeline" links to target
  "Boost::thread" but the target was not found.  Perhaps a find_package()
  call is missing for an IMPORTED target, or an ALIAS target is missing?
```
and more.

Reply:
I suggest to put that line in `CGAL_pointmatcher_support.cmake`.

Actually, it should be also upstreamed in `libpointmatcherConfig.cmake`.